slsim/Util/param_util.py

```diff
@@ -256,3 +256,15 @@ def deg2_to_cone_angle(solid_angle_deg2):
     solid_angle_sr = solid_angle_deg2 * (np.pi / 180) ** 2
     theta = np.arccos(1 - solid_angle_sr / (2 * np.pi))  # rad
     return theta
+
+
+def ellipticity_slsim_to_lenstronomy(e1_slsim, e2_slsim):
+    """Converts ellipticity component from slsim convension to lenstronomy convention.
```

Comment:
do you also want to mention the SLSim conventions and the lenstronomy conventions? Since you know them both it's worth writing them down

Reply:
I mentioned both slsim and lenstronomy convention for position angle direction.